src/js/views/globals/search/patient-search_views.js

```diff
@@ -10,8 +10,21 @@ import Picklist from 'js/components/picklist';
 import './patient-search.scss';
 
 const TipTemplate = hbs`
-<div>{{ @intl.globals.search.patientSearchViews.picklistEmptyView.searchTip }}</div>
-<div>{{fas "keyboard"}}<strong class="u-margin--l-8">{{ @intl.globals.search.patientSearchViews.picklistEmptyView.shortcut }}</strong></div>
+  <div class="patient-search__tip">{{ @intl.globals.search.patientSearchViews.picklistEmptyView.searchTip }}</div>
+  <div class="patient-search__search-by">
+    <div class="patient-search__search-by-title">Or search for patients by</div>
+    <div class="u-margin--t-8 qa-patient-search__search-by-option">
```

Comment:
I need to target this element in the tests. I was using the `patient-search__search-by-option` class name before.

Does using a `qa-patient-search__search-by-option` work instead (with the `qa-` prefix)?

We discussed that here: https://app.shortcut.com/roundingwell/story/30275/update-test-selectors-for-best-practices. But we haven't implemented it in the app yet.

Reply:
yeah though I don't think it needs to be BEM'd.  `.qa-something-something` is fine.  it really doesn't even need to be very unique because we should be scoping tests down such that something as simple as `.qa-item` would suffice... similar to `.js-select` etc..  I don't think it should necessarily be _that_ generic, but.. I dunno.. `.qa-search-option` or something would work.

Reply:
Cool, makes sense. Just updated those to `.qa-search-option`.